commands/replace_test.go

```diff
@@ -0,0 +1,59 @@
+package commands
+
+import (
+	"github.com/Shopify/themekit"
+	"github.com/Shopify/themekit/theme"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
+)
+
+func TestFullReplace(t *testing.T) {
```

Comment:
Added these tests to figure out why replace was not working as intended, thinking there was a bug in that code. After I wrote all these tests I couldn't find anything wrong with it. Might as well keep them now. :)